Parsers/VSISDMI.cs

```diff
@@ -2282,6 +3132,80 @@ public VSISDMI()
             }
         };
 
+        public static BitField MMI_STM_L_VALUE => new BitField
+        {
+            Name = "MMI_STM_L_VALUE",
+            BitFieldType = BitFieldType.UInt16,
+            Length = 16,
+            Comment = "Length of X_VALUE for default value (range: 0-10)."
+        };
+
+        public static BitField MMI_STM_L_VALUE2 => new BitField
+        {
+            Name = "MMI_STM_L_VALUE",
+            BitFieldType = BitFieldType.UInt16,
+            Length = 16,
+            Comment = "Length of X_VALUE for pick-up list value (range: 0-10)."
+        };
+
+        public static BitField MMI_STM_NID_DATA => new BitField
+        {
+            Name = "MMI_STM_NID_DATA",
+            BitFieldType = BitFieldType.UInt8,
+            Length = 8,
+            Comment = "Identifier of a Specific STM Data to be entered."
+        };
+
+        public static BitField MMI_STM_Q_FOLLOWING => new BitField
+        {
+            Name = "MMI_STM_Q_FOLLOWING",
+            BitFieldType = BitFieldType.Bool,
+            Length = 1,
+            Comment = "Indicate following data to be viewed at the same time"
+        };
+
+        // TODO Johan, please check whether the length is required along with the scaling factor
```

Comment:
No the length will be ignored when VariableLength is set. You can look in BitStreamParser.cs in s-tools to see the logic

Reply:
Oh yes, i remember now. I guess best to leave it in for clarity though.